README.md

```diff
@@ -37,6 +37,48 @@ or add
 
 to the `require` section of your `composer.json` file.
 
+## Usage with Queue utility
+
+1. Copy configuration file `./vendor/yiisoft/yii-queue/bin/QueueContainer.php` to `root` folder of your project.
```

Comment:
Suggest require the user to create `yii-queue-params.php` in root directory with queue params only and don't require copy full container configuration. It's more cleary...